infra/aws/terraform/accounts/spire/main.tf

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2022 The CNCF Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+variable "user-email-addresses" {
+  type = list(string)
+  default = [
+    "azdagron@gmail.com",
+    "ryan.turner253@icloud.com",
+    "amartinezfayo@gmail.com",
+    "evan2645@gmail.com",
+    "marcosyacob@gmail.com"
```

Comment:
It's odd to see these specified here, rather than at the place where the module is called.

Reply:
Keeping all of the values at the top and passing them through when the modules are called for laying out the document easier